### ISSUE TITLE
Missing Functions

### ISSUE THREAD

**Issue by DTV96Calibre** [mveteanu/p5.SceneManager#1]
mouseClicked and windowResized aren't handled by the scene manager, likely among other functions. I've temporarily worked around this in my own project by using mgr.scene.oScene.mouseClicked() to reference the function in the scene object. I'll probably try adding support for these functions later when I've looked into it more.

**Comment by PixsaOJ** [comment]
Close this please

**Comment by wmpowell8** [comment]
> Close this please

I agree; SceneManager handles events differently now, in a DRYer manner. I'm working on a pull request to fix this issue. Also, `mouseClicked` wiring has been added; `windowResized` is what I'll add in my pull request.

**Comment by wmpowell8** [comment]
I just made the pull request (see above)
